### PR TITLE
Show check identifiers at multi-check rules

### PR DESCRIPTION
**Pull request by jan-cerny**
Shows OVAL Definition ID and Title when an XCCDF rule uses
multi-check='true'. The multi-check is used in rule
security_patches_up_to_date in SCAP 1.3 datastreams. It wasn't possible
to see which vulnerabilities have been found because all the checks have
the same XCCDF rule ID.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1771438

Fixes: #1320

### CODE REVIEW

**Comment by jan-cerny** [comment]
We need to update the HTML report as well, see https://github.com/ComplianceAsCode/content/issues/5037

**Comment by matejak** [comment]
test this please

File: xsl/xccdf-report-oval-details.xsl [hunk 1]
```diff
@@ -63,6 +65,30 @@ Authors:
 </xsl:template>
 
 <xsl:template mode='brief' match='ovalres:definition'>
+    <xsl:variable name="definition" select="key('ovaldef-definition', ./@definition_id)" />
```
Comment (jan-cerny) [inline xsl/xccdf-report-oval-details.xsl]: @jan-cerny Please submit this change as a separate PR, it isn't mandatory part of the fix.

**Comment by jan-cerny** [comment]
This is now ready for a review. You can check the example stdout and HTML report from evaluation of PCI-DSS profile on a RHEL 8 machine:
[pci-dss-results.zip](https://github.com/OpenSCAP/openscap/files/3943778/pci-dss-results.zip)
Focus on rule "Ensure Software Patches Installed".

**Comment by matejak** [comment]
test this please

**Comment by evgenyz** [comment]
@jan-cerny Would it affect Workbench? Does it have to be patched as well?

**Comment by jan-cerny** [comment]
@evgenyz You're great! Thanks for pointing this out. I played with this Unfortunately, we need to add support for the multi-check feature in SCAP Workbench as well. Currently, SCAP Workbench shows only 1 of the N results of the checks (probably the latest evaluated check result).  I tried to compile and run Workbench with the patched OpenSCAP and I have found that this patch neither improves it nor breaks it, as it's already broken. I assume that to integrate this feature in SCAP Workbench we also need to add additional column to the `--progress` output of `oscap xccdf eval` or some other way to distinguish between the checks within a single XCCDF rule, which means changes on OpenSCAP side.

**Comment by jan-cerny** [comment]
I checked SCAP Workbench code. Unfortunately, I haven't found any easy way to bring this feature to SCAP Workbench. I have reported an issue against Workbench about that in https://github.com/OpenSCAP/scap-workbench/issues/242.

I suggest not blocking this PR by SCAP Workbench. As I said in my previous comment, this patch neither fixes nor breaks the Workbench. That's mainly because SCAP Workbench uses `oscap` in `--progress` mode, but this patch modifies the normal output mode.

Please let me know your thoughts about the improvements in the HTML report and check also if the stdout format is OK for you.

**Comment by evgenyz** [comment]
The patch set looks good to me. We will deal with the Workbench in a separate PR. HTML report is definitely makes more sense now, and as stdout is not supposed to be used for inter-op we can allow a a bit of swag in any direction.
